sqlmesh/core/model/definition.py

```diff
@@ -704,6 +701,8 @@ class SeedModel(_Model):
 
     kind: SeedKind
     seed: Seed
+    column_hashes_: t.Optional[t.Dict[str, str]] = Field(default=None, alias="column_hashes")
+    is_hydrated: bool = True
```

Comment:
does this get serialized? should this be private and only exposed in a property?

Reply:
It does get serialized, why shouldn't it?

Reply:
i was thinking in the db, but i guess it makes sense over the wire